src/content/News.md

```diff
@@ -1,3 +1,20 @@
-# News
+# MIND News
+Apps.digitalpsych.org is officially MIND - the Mhealth Index and Navigation Database. The mission of MIND is to make mental health tools more accessible through connecting users with tools that suit their unique needs and preferences. From presentations to partnerships, check out how MIND is working to advance digital mental health. 
 
-There are no new annoncements at this time.
\ No newline at end of file
+# Partnership with BridgingApps 
+BridgingApps is a website of Easter Seals Greater Houston. The Easter Seals network is the largest provider of disability services in North America, outside of the government. Created in 2010, BridgingApps aims to help parents of children with disabilities shortcut the process of finding appropriate apps to help their children learn, progress in therapy, and enhance their lives.  With over 3000 apps now represented on the database, Bridging Apps serves children, adults, caregivers, veterans, among other populations. Check out more about the collaboration here: https://www.bridgingapps.org/2020/10/bridgingapps-collaborates-with-harvard-digital-psychiatry-team-to-highlight-new-tool-for-mental-health-apps/
```

Comment:
Same here, you can cleanup the link address if you want.  See note below for example.